test/util/test_helpers.py

```diff
@@ -72,42 +83,39 @@ def assert_variable_set_fails(self, variable_id, error_class, exit_code=0):
     with self.assertRaises(error_class):
         self.set_variable(variable_id, uuid.uuid4().hex, exit_code)
 
+
 def print_instead_of_raise_error(self, variable_id, error_message_regex):
     output = self.invoke_cli(self.cli_auth_params,
                              ['variable', 'set', '-i', variable_id, '-v', uuid.uuid4().hex], exit_code=1)
 
     self.assertRegex(output, error_message_regex)
 
- # *************** POLICY ***************
 
-def generate_policy_string(self):
-        variable_1 = 'simple/basic/{}'.format(uuid.uuid4().hex)
-        variable_2 = 'simple/space filled/{}'.format(uuid.uuid4().hex)
-        variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
+# *************** POLICY ***************
 
-# this policy is purposefully not indented because
-# the """ formatter is indentation sensitive and
-# policy should start at the far left of the line
-        policy = \
-"""
-- !variable
-  id: {variable_1}
-- !variable
-  id: {variable_2}
-- !variable
-  id: {variable_3}
-"""
+def generate_policy_string():
+    variable_1 = 'simple/basic/{}'.format(uuid.uuid4().hex)
+    variable_2 = 'simple/space filled/{}'.format(uuid.uuid4().hex)
+    variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
-        dynamic_policy = policy.format(variable_1=variable_1,
-                                       variable_2=variable_2,
-                                       variable_3=variable_3)
+    # We purposefully build this policy like this
```

Comment:
We purposefully construct the policy like this. Otherwise, Intelj's auto-indent keeps changing the string indentation and breaking all tests that use this function